src/main/java/org/example/Singleton.java

```diff
@@ -0,0 +1,20 @@
+package org.example;
+
+public final class Singleton {
+
+    private static Singleton INSTANCE;
+
+    private Singleton() {
+    }
+
+    public static Singleton getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new Singleton();
+        }
+        return INSTANCE;
+    }
+
+    public void connectToDatabase() {
+        System.out.println("Connecting to database");
+    }
+}
```

Comment:
You don't need this class, I created it just for the purpose of showing and later deleted it